README.md

```diff
@@ -57,4 +57,68 @@ bool transmitCanFrame(CanardFrame const & frame) {
 ```
 
 ### Contribution
-Please take a look at the [wiki](https://github.com/107-systems/107-Arduino-UAVCAN/wiki) for notes pertaining development of `107-Arduino-UAVCAN`.
+#### How to add missing wrappers
+##### Step 1) Generate C header files from DSDL
+**Option A) Use [nunavut/nnvg](https://github.com/UAVCAN/nunavut)**
+* Install **nunavut**
+```bash
+git clone https://github.com/UAVCAN/nunavut && cd nunavut
+python3.8 -m pip install .
+```
+* Get some DSDL
+```bash
+git clone https://github.com/UAVCAN/public_regulated_data_types
+```
+* Generate C header files from DSDL
+```bash
+nnvg --target-language c \
+     --pp-max-emptylines=1  \
+     --pp-trim-trailing-whitespace \
+     --target-endianness=little \
+     --enable-serialization-asserts \
+     --outdir public_regulated_data_types/uavcan-header \
+     public_regulated_data_types/uavcan
+
+nnvg --target-language c \
+     --pp-max-emptylines=1  \
+     --pp-trim-trailing-whitespace \
+     --target-endianness=any \
```

Comment:
```suggestion
     --target-endianness=little \
```

Above you are using `little`. Are there any non-little-endian platforms that Arduino runs on? Endianness-invariant code is usually larger and slower.

Reply:
Good catch :+1: Right now the library has only be run on little endian architectures (ARM, x86, EXP32). However, this might change in the future. Since I have no influence or even knowledge for which future platforms Arduino cores might be developed I prefer to support any sort of endianness.